src/vs/workbench/contrib/notebook/browser/controller/editActions.ts

```diff
@@ -157,6 +160,37 @@ registerAction2(class DeleteCellAction extends NotebookCellAction {
 			return;
 		}
 
+		let confirmation: IConfirmationResult;
+		const notebookExecutionStateService = accessor.get(INotebookExecutionStateService);
+		const runState = notebookExecutionStateService.getCellExecution(context.cell.uri)?.state;
+		const configService = accessor.get(IConfigurationService);
+
+		if (runState === NotebookCellExecutionState.Executing && configService.getValue(NotebookSetting.confirmDeleteRunningCell)) {
+			const dialogService = accessor.get(IDialogService);
+			const primaryButton = localize('confirmDeleteButton', 'Confirm Delete');
+
+			confirmation = await dialogService.confirm({
+				type: 'question',
+				message: localize('confirmDeleteButtonMessage', 'Are you sure you want to delete a running cell?'),
```

Comment:
"This cell is running, are you sure you want to delete it?"

Reply:
Thanks! my english is not good haha